### PR TITLE
Implementing a voting for LIP-13 EasyTrack payment limits 

### PR DESCRIPTION
**Pull request by folkyatina**
This a early draft implementation of the voting that implement the LIP with EasyTrack payment limits

See [LIP](https://hackmd.io/EfKZ-fKmRCWzDC_mU2zk0A?view) for details

### CODE REVIEW

**Comment by krogla** [comment]
LGTM

File: utils/permission_parameters.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    condition32 = convert.to_uint(condition, 'uint32')
+    success32 = convert.to_uint(success, 'uint32')
+    failure32 = convert.to_uint(failure, 'uint32')
+
+    value = condition32 + (success32 << 32) + (failure32 << 64)
+
+    return ArgumentValue(convert.to_uint(value, 'uint240'))
+
+
+def _to_uint240(val: Union[int, str]) -> int:
```
Comment (TheDZhon) [inline utils/permission_parameters.py:97]: Maybe add some pre-conditions (asserts) here to check? 

All magical passes with bytes become more secure.

File: tests/test_setup_easytrack_limits.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+def test_permission_fails_for_usdc(acl, finance, ldo_holder, amount):
+    assert not has_payments_permission(acl, finance, evmscriptexecutor, usdc_token, ldo_holder.address, amount)
+    with brownie.reverts('APP_AUTH_FAILED'):
+        finance.newImmediatePayment(usdc_token, ldo_holder.address, amount, 'Should be reverted',
+                                    {'from': evmscriptexecutor})
+
+
+@pytest.mark.parametrize('token', [eth, steth, ldo, dai])
+@given(amount=strategy('uint', min_value=1))
+def test_permission_fails_for_other_sender(acl, finance, accounts, token, amount):
```
Comment (TheDZhon) [inline tests/test_setup_easytrack_limits.py:136]: maybe also add another one perm check for another legit sender to show that parameters not applied? I mean `voting` here.

**Comment by TheDZhon** [comment]
I like the way it's implemented, you are doing great! :+1: 
Another like for the hypothesis testing :fire: 

Please have a look at the above comments (they are non-blockers, of course)

**Pull request merged**